packages/web-components/fast-components/CONTRIBUTING.md

```diff
@@ -4,17 +4,51 @@
 
 The component explorer is a [site](https://explore.fast.design) that showcases the components in the `@microsoft/fast-components` package. This acts as part of the documentation story, in order to expose a component it must have a [definition](#definition) and [configuration](#configuration).
```

Comment:
You'll want to update this link

Reply:
@nicholasrice What is the correct link?

Reply:
@nicholasrice can you provide the link?  I'm not sure which we're talking about.  But, would be great to get this in since it's gone stale.  Thanks